sdk/appconfiguration/test-resources.json

```diff
@@ -48,16 +48,30 @@
             "defaultValue": "azconfig-net",
             "type": "string"
         },
-        "endpointSuffix": {
+        "azConfigEndpointSuffix": {
             "defaultValue": ".azconfig.io",
             "type": "string"
+        },
+        "keyVaultEndpointSuffix": {
+            "defaultValue": ".vault.azure.net",
+            "type": "string",
+            "metadata": {
+                "description": "Endpoint suffix for sovereign clouds, requies the preceeding '.'. The default uses the public Azure Cloud (.vault.azure.net)"
```

Comment:
Misspell "requies" should be "requires", and "preceeding" should be "proceeding"